base/fft/FFTW.jl

```diff
@@ -101,16 +101,16 @@ alignment_of(A::FakeArray) = Int32(0)
 # FFTW's api/import-wisdom-from-file.c file].
 
 function export_wisdom(fname::AbstractString)
-    f = ccall(:fopen, Ptr{Void}, (Cstring,Ptr{UInt8}), fname, "w")
+    f = ccall(:fopen, Ptr{Void}, (Cstring,Cstring), fname, "w")
```

Comment:
OK, will be changing to symbols where appropriate, and back to `Ptr{UInt8}` otherwise. The IR generated by the `Cstring` conversions is remarkably long indeed, but performance seemed to be okay. Will have a closer look.